pkgs/applications/terminal-emulators/wezterm/default.nix

```diff
@@ -81,11 +82,14 @@ rustPlatform.buildRustPackage rec {
     CoreGraphics
     Foundation
     libiconv
+    System
     UserNotifications
   ];
 
   buildFeatures = [ "distro-defaults" ];
 
+  env.NIX_LDFLAGS = lib.optionalString stdenv.isDarwin "-framework System";
```

Comment:
Does the build still succeed without this line on your machine?

Reply:
No, it doesn't. I actually tried first without the line cause ld has `-lSystem` flag passed so it *should've worked*, but didn't work for me.